chirp/drivers/ft450d.py

```diff
@@ -939,13 +1002,12 @@ def _set_memory(self, mem, _mem):
             else:
                 setattr(_mem, setting.get_name(), setting.value)
 
-
     @classmethod
     def match_model(cls, filedata, filename):
         """Match the opened/downloaded image to the correct version"""
         if len(filedata) == cls.MEM_SIZE + 7:    # +7 bytes of model name
             rid = filedata[cls.MEM_SIZE:cls.MEM_SIZE + 7]
-            if rid.startswith(cls.MODEL.encode()):
+            if rid.startswith(cls.MODEL):

```

Comment:
`filedata` is a `bytes` here, so you can't compare it with `cls.MODEL`, which is a string. You're failing the detect test as a result...